khmer/__init__.py

```diff
@@ -86,6 +86,13 @@
 del get_versions
 
 
+buckets_per_byte = {
```

Comment:
`_buckets_per_byte`? Just to prevent people from being tempted to modify values here.